integration-tests/Tests/IntegrationTests/PrimitiveTests.swift

```diff
@@ -0,0 +1,28 @@
+import RustLib
+import XCTest
+
+@testable import SharedLib
+
+/// Tests for generic types such as `type SomeType<u32>`
+class PrimitiveTests: XCTestCase {
+    /// Run tests where Rust calls Swift functions that take primitive args.
+    func testRustCallsSwiftPrimitives() throws {
+        test_rust_calls_swift_primitives()
+    }
+
+    /// Run tests where Swift calls Rust functions that take primitive args.
+    func testSwiftCallsRustPrimitives() throws {
```

Comment:
In the old test suite you could click on a test in Xcode to run just that test.

In this new test suite, let's leave a comment above each test with the command to run just that test.

````
/// ## To Run
/// ```
/// swift test .... args for running just this test here ...
/// ```
````


Additionally, let's also add a command above the `class PrimitiveTests` that can be used to run all the primitive tests.

The idea is for a contributor to be able to easily figure out how to run the tests for the part of the system that they are working on.

Currently, most contributors have more Rust experience than Swift experience, so more hand-holding on the Swift side is useful.